pkg/proxy/userspace/proxier.go

```diff
@@ -687,12 +696,16 @@ func iptablesCommonPortalArgs(destIP net.IP, destPort int, protocol api.Protocol
 		args = append(args, "-d", fmt.Sprintf("%s/32", destIP.String()))
 	}
 
+	if excludeContainerSourceTraffic {
+		args = append(args, "-m", "physdev", "--physdev-is-in")
```

Comment:
I need "!" if use "-j DROP". (negative filter)
I did not need to use "!" because I send the packet to proxier on the physdev-is-in match (-j REDIRECT). (positive filter)